srml/fees/src/lib.rs

```diff
@@ -47,6 +69,11 @@ decl_module! {
 
 		fn on_finalise() {
 			let extrinsic_count = <system::Module<T>>::extrinsic_count();
+			let block_fee = (0..extrinsic_count).fold(<AssetOf<T>>::sa(0), |total, index| {
```

Comment:
Can we not combine this with the loop below? To have this loop just once?

Reply:
@bkchr all updated :]